docs/src/rsh/compute/index.md

```diff
@@ -618,9 +618,21 @@ band(a, b)    // &
 bior(a, b)    // |
 polyEq(a, b)  // ==, ===
 polyNeq(a, b) // !=, !==
+safeAdd(a, b)
+safeSub(a, b)
+safeMul(a, b)
+safeDiv(a, b)
+safeMod(a, b)
+veriAdd(a, b)
+veriSub(a, b)
+veriMul(a, b)
+veriDiv(a, b)
+veriMod(a, b)
 ```
 
 All binary expression operators have a corresponding named function in the standard library.
+The named arithmetic functions prefixed with `{!rsh} safe`, e.g. `{!rsh} safeAdd`, bypass `{!rsh} verifyArithmetic` and are not statically verified; they are only checked at runtime.
+The named arithmetic functions prefixed with `{!rsh} veri`, e.g. `{!rsh} veriAdd`, are statically verified, regardless of `{!rsh} verifyArithmetic`; no runtime checks will be generated for these verified operations.
```

Comment:
I like the way you expressed this. I think you should also make a comment on the option that you can get individual function level verification when you want it